apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html

```diff
@@ -1549,10 +1562,15 @@ <h5>
                                                     [ngClass]="{ 'dropup-menu': isDropUp }"
                                                 >
                                                     <ul class="select-action-dropdown">
+                                                        <li (click)="onPerformAction(item, 'paid')" *ngIf="selectedVoucher === 'debit note' || selectedVoucher === 'credit note'">
```

Comment:
Make enum file if not then make constant in enum

Reply:
Not use Enum in HTML